src/components/result-view-tab.js

```diff
@@ -63,21 +74,32 @@ export const ResultViewTab = ({
 
     return (
         <Paper className={clsx('singlestretch-child', classes.table)}>
-            <Tabs
-                value={tabIndex}
-                onChange={(event, newTabIndex) => setTabIndex(newTabIndex)}
-            >
-                <Tab
-                    label={intl.formatMessage({
-                        id: 'loadFlowResults',
-                    })}
-                />
-                <Tab
-                    label={intl.formatMessage({
-                        id: 'securityAnalysisResults',
-                    })}
-                />
-            </Tabs>
+            <div className={classes.div}>
+                <Tabs
+                    value={tabIndex}
+                    onChange={(event, newTabIndex) => setTabIndex(newTabIndex)}
+                >
+                    <Tab
+                        label={intl.formatMessage({
+                            id: 'loadFlowResults',
+                        })}
+                    />
+                    <Tab
+                        label={intl.formatMessage({
+                            id: 'securityAnalysisResults',
+                        })}
+                    />
+                </Tabs>
+                {!allowModificationsOnNode(workingNode, selectedNode) &&
+                    selectedNode?.type !== 'ROOT' && (
```

Comment:
Same as before, maybe use a function whose name makes sense with the warning displayed.

Reply:
Done